x-pack/plugins/observability_onboarding/server/routes/flow/route.ts

```diff
@@ -7,113 +7,22 @@
 
-const logMonitoringPrivilegesRoute = createObservabilityOnboardingServerRoute({
-  endpoint: 'GET /internal/observability_onboarding/custom_logs/privileges',
-  options: { tags: [] },
-
-  handler: async (resources): Promise<{ hasPrivileges: boolean }> => {
-    const { context } = resources;
-
-    const {
-      elasticsearch: { client },
-    } = await context.core;
-
-    const hasPrivileges = await hasLogMonitoringPrivileges(
-      client.asCurrentUser
-    );
-
-    return { hasPrivileges };
-  },
-});
-
-const installShipperSetupRoute = createObservabilityOnboardingServerRoute({
-  endpoint:
-    'GET /internal/observability_onboarding/custom_logs/install_shipper_setup',
-  options: { tags: [] },
-  async handler(resources): Promise<{
-    apiEndpoint: string;
-    scriptDownloadUrl: string;
-    elasticAgentVersion: string;
-  }> {
-    const { core, plugins, kibanaVersion } = resources;
-    const coreStart = await core.start();
-
-    const kibanaUrl =
-      core.setup.http.basePath.publicBaseUrl ?? // priority given to server.publicBaseUrl
-      plugins.cloud?.setup?.kibanaUrl ?? // then cloud id
-      getFallbackKibanaUrl(coreStart); // falls back to local network binding
-    const scriptDownloadUrl = `${kibanaUrl}/plugins/observabilityOnboarding/assets/standalone_agent_setup.sh`;
-    const apiEndpoint = `${kibanaUrl}/internal/observability_onboarding`;
-
-    return {
-      apiEndpoint,
-      scriptDownloadUrl,
-      elasticAgentVersion: kibanaVersion,
-    };
-  },
-});
-
-const createApiKeyRoute = createObservabilityOnboardingServerRoute({
-  endpoint: 'POST /internal/observability_onboarding/custom_logs/save',
-  options: { tags: [] },
-  params: t.type({
-    body: t.type({
-      name: t.string,
-      state: t.record(t.string, t.unknown),
-    }),
-  }),
-  async handler(
-    resources
-  ): Promise<{ apiKeyEncoded: string; onboardingId: string }> {
-    const {
-      context,
-      params: {
-        body: { name, state },
-      },
-      core,
-      request,
-    } = resources;
-    const coreStart = await core.start();
-    const {
-      elasticsearch: { client },
-    } = await context.core;
-    const { encoded: apiKeyEncoded } = await createShipperApiKey(
-      client.asCurrentUser,
-      name
-    );
-
-    const savedObjectsClient = coreStart.savedObjects.getScopedClient(request);
-
-    const { id } = await saveObservabilityOnboardingState({
-      savedObjectsClient,
-      observabilityOnboardingState: {
-        state: state as ObservabilityOnboardingState['state'],
-        progress: {},
-      },
-    });
-
-    return { apiKeyEncoded, onboardingId: id };
-  },
-});
-
-const updateOnboardingStateRoute = createObservabilityOnboardingServerRoute({
-  endpoint:
-    'PUT /internal/observability_onboarding/custom_logs/{onboardingId}/save',
+const updateOnboardingFlowRoute = createObservabilityOnboardingServerRoute({
+  endpoint: 'PUT /internal/observability_onboarding/flow/{onboardingId}/save',
```

Comment:
since we're updating the routes names, we could leave off the `/save` and `/create` at the end of the routes, since those actions are implied by the HTTP methods (PUT and POST).